### PR TITLE
Query CartoDB for tract ids within an area

### PR DESCRIPTION
**Pull request by sharph**

### CODE REVIEW

**Comment by sharph** [comment]
With this PR if you select an area with the circle tool or the polygon tool a list of tract ids within county and state ids will appear in the console.

The ACS module is intended to handle searching for tracts as well as requesting and transforming the data from the census api.

File: app/scripts/config.js [hunk 1]
```diff
@@ -19,7 +19,8 @@
         cartodb: {
             visId: 'a4ed4438-ffec-11e4-8b9a-0e9d821ea90d',
             account: 'azavea-demo',
-            tableName: 'mudf15q1int'
+            tableName: 'mudf15q1int',
+            acsTableName: 'seattle_area_tracts'
```
Comment (CloudNiner) [inline app/scripts/config.js]: Could this more clearly be named `tractsTableName`?

File: app/scripts/acs/acs-service.js [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+        return module;
+
+        /**
+         * Returns a where clause that selects a radius around a point.
+         * Return value contains {geom} token to be replaced with geometry
+         * column name.
+         * @param {Number} x
+         * @param {Number} y
+         * @param radius
```
Comment (CloudNiner) [inline app/scripts/acs/acs-service.js]: Note units

File: app/scripts/acs/acs-service.js [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+            var dfd = $q.defer();
+            cartodbsql.execute(query).done(function (data) {
+                dfd.resolve(data.rows);
+            }).error(function (error) {
+                dfd.reject(error);
+            });
+            return dfd.promise;
+        }
+
+        function strFormat(str, params) {
```
Comment (CloudNiner) [inline app/scripts/acs/acs-service.js]: Isn't strFormat already in a Utils module somewhere? It's also used in the other cartodb service right? If so, might want to use that one or just put it in its own Utils service (possibly along with makeRequest)?

**Comment by CloudNiner** [comment]
+1, consider minor cleanup

**Pull request merged**